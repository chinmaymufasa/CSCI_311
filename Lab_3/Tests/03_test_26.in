2
167
5
2
86
5
2
140
5
2
91
5
2
60
5
1
5
-1