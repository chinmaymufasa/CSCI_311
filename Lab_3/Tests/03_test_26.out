167 
86 167 
86 140 167 
86 91 140 167 
60 86 91 140 167 
Not found
